developerportal/apps/content/templates/content.html

```diff
@@ -6,18 +6,32 @@
 {% block body_class %}content{% endblock %}
 
 {% block content %}
-  <main>
+  <main class="{% if page.hero_image %}content-has-hero{% endif %}">
     <div class="base-grid">
       <section>
-        <h1>{{ page.title }}</h1>
-        {% for block in page.body %}
-          {% if block.block_type == 'image' %}
-            {% image block.value width-672 as rendition %}
-            {% include "atoms/image.html" with image=block.value rendition=rendition %}
-          {% else %}
-            {{ block }}
-          {% endif %}
-        {% endfor %}
+        {% if page.hero_image %}
+          {% image page.hero_image width-1024 as hero %}
```

Comment:
Do we need to support retina images here? If so, maybe this is a good opportunity to use `srcset` since serving a 2048px wide image to a mobile device wouldn't be cool.

Reply:
To be addressed as separate responsive image ticket